ext/oss/sink.go

```diff
@@ -33,6 +37,8 @@ type OSSSink struct {
 	batchSize              int
 	enableOverwrite        bool
 	skipHeader             bool
+
+	partiallyUploadedFiles map[string]int // map of destinationURI to the sequence of file being uploaded
```

Comment:
can we rename it as `partialFileRecordCounters` instead?

Reply:
agree